Kconfig

```
@@ -1023,6 +1023,9 @@ menu "LVGL configuration"
 		config LV_USE_PNG
```

Please rename it to make it cleaner which PNG library is which. Also add a few lines of description to make it easier to decide which one to choose.
```suggestion
		config LV_USE_LODEPNG
```

---

Can I submit another PR to rename `lv_png` -> `lv_lodepng`?

---

Sure! Can I merge this one?